waspc/data/Generator/templates/sdk/wasp/auth/session.ts

```diff
@@ -19,7 +19,7 @@ export async function createSession(authId: string): Promise<Session> {
 
 // PRIVATE API
 export async function getSessionAndUserFromBearerToken(req: ExpressRequest): Promise<{
-  user: AuthUser | null,
+  user: AuthUserData | null,
   session: Session | null,
 }> {
```

Comment:
I know it's not related to this PR, but I was just wondering, why isn't the return type here:
```typescript
Promise<{ user: AuthUser, session: Session } | null>
```
This type would be more correct, as `{ user: AuthUser | null, session: Session | null }` implies you could have several states that in reality aren't possible (e.g., ` { user: <some user> , session: null }`).

Is it because of how the frontend code usually uses this response?

Reply:
I kinda went with Lucia's types: https://github.com/lucia-auth/lucia/blob/d285a08dc2eda428236a1b267b19c01886e786ad/packages/lucia/src/core.ts#L122

But ... I can adjust them to be more sane 👍